turtlebot2:
    task_and_robot_environment_name: 'MyTurtleBot2Maze-v0'
    ros_ws_abspath: "/home/pierre/simulation_ws"
    running_step: 0.04 # amount of time the control will be executed
    pos_step: 0.016     # increment in position for each command
